.github/CHANGELOG.md

```diff
@@ -123,6 +123,9 @@
 
 ### Documentation
 
+* Update Lightning-Tensor installation docs and usage suggestions.
+  [(#979)](https://github.com/PennyLaneAI/pennylane-lightning/pull/979)
```

Comment:
Hi @AntonNI8, what's the status of this PR?

Reply:
Hey @maliasadi , I just wanted to make some tweaks to the installation instructions for the lightning devices, but figured it would be best to wait until all of the v0.39 release work had been completed.
Pending reviews, I think we can go ahead with it.